src/style/_globals.ts

```diff
@@ -7,6 +7,7 @@ export const skyBlueColor = '#00aaff';
 export const oceanBlueColor = '#0088cc';
 //white-Gray base collection
 export const shadowColor = '#4c4c4c';
+export const textViewHeader = '#5a5b5e';
```

Comment:
?

Reply:
sorry, wrong naming, need to be `textViewHeaderColor` , the color is given from zeppelin

Reply:
sorry, wrong naming, need to be `textViewHeaderColor` , the color is given from zeppelin

Reply:
Don not name color by functionality. Instead - use color convention made by @9Syncopation